agent/api/container.go

```diff
@@ -108,7 +109,7 @@ type Container struct {
 	SentStatusUnsafe ContainerStatus `json:"SentStatus"`
 	sentStatusLock   sync.RWMutex
```

Comment:
Why keep this one? Can we use `lock` for protecting `SentStatusUnsafe ` as well?